pages/linux/lsattr.md

```diff
@@ -0,0 +1,23 @@
+# lsattr
+
+> List file attributes on a Linux file system.
+
+- Display the attributes of the files in the current directory:
+
+`lsattr`
+
+- List the attributes of files in {{path}}:
```

Comment:
We usually do not repeat tokens in description. It is probably better to say
- List the attributes of files in a given path: